src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -87,6 +87,28 @@ public static class CLIOptions
             JsonKey = "project-file"
         };
 
+        public static readonly CLIOption<string> CoverageAnalysis = new CLIOption<string>
+        {
+            ArgumentName = "--coverageAnalysis",
+            ArgumentShortName = "-ca <mode>",
+            DefaultValue = "off",
+            ArgumentDescription = @"Use coverage info to speed up execution. Possible values are: off, all, perTest.
+    - off: coverage data is not captured (default mode)
+    - all: capture the list of mutants covered by the test. Test only this mutants; non covered ones are assumed as survivors.
+    - perTest: capture the list of mutants covered by each test. Only the tests that cover a particular mutant are tested for each one. Only supported by vsTest runner for now",
+            JsonKey = "coverageAnalysis"
+        };
+
+        public static readonly CLIOption<string[]> AdvancedOptions = new CLIOption<string[]>()
+        {
+            ArgumentName = "--advancedOptions",
```

Comment:
Actually, I have several in mind, such as:
- ordering strategy: since we can abort run on the first failed test, we could run the fastest first
or the most specific test
- dev mode could be migrated here
- using pipe or environment variable for coverage capture
- activate some debug mode
....

Reply:
But still I see no advantage of creating an advanced option. These could simply be options on stryker that we can document as "advanced". And also I'd like to keep stryker as simple as it could be to use. Extra options could only make beginners with mutation testing confused.

And for the ordering, that seems a bit like micro optimalisation. It could prove useful but I would suggest enabling that by default and make it non configurable. Same for pipe or environment coverage capture.

Reply:
I concur on the idea of having the interface as straight forward as possible, hence my original signature of _mode:fast_ to enable both coverage analysis and abort on test fail.

the idea of advanced is that their are not for beginners 😉. In my mind, those extra options are used to tune the behaviour for people with order sensitive tests, for example.

anyway, I understand this does not match your vision of your product, so I will transform to dedicated switch

Reply:
By the way, now that you understand that 'OptimizationFlags' are flags indeed, does this naming make more sense to you? the 'CaptureCoveragePerTest' option can resolve coverage capture issues for some multi threaded project. But it is not available now.

Reply:
migrated to abort-test-on-fail